pnpm-lock.yaml

```diff
@@ -2929,7 +2929,7 @@ packages:
       '@types/node': '*'
       cosmiconfig: '>=7'
       ts-node: '>=10'
-      typescript: ^5
+      typescript: '>=4'
```

Comment:
I'm not sure why this PR would affect this. Could you make sure you are using `pnpm@8.7.0` with no mirror environment?